rustyms-py/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "rustyms-py"
+version = "0.8.1"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[lib]
+name = "rustyms_py"
```

Comment:
Does it make sense to have kebab case here as well? Or was this underscore needed?

Reply:
Apparently it is needed:

```
rust-analyzer failed to load workspace: Failed to load the project at C:\Users\ralfg\git\rustyms\rustyms-py/Cargo.toml: Failed to read Cargo metadata from Cargo.toml file C:\Users\ralfg\git\rustyms\rustyms-py/Cargo.toml, Some(Version { major: 1, minor: 76, patch: 0 }): Failed to run `"cargo" "metadata" "--format-version" "1" "--manifest-path" "C:\\Users\\ralfg\\git\\rustyms\\rustyms-py/Cargo.toml" "--filter-platform" "x86_64-pc-windows-msvc"`: `cargo metadata` exited with an error: error: failed to parse manifest at `C:\Users\ralfg\git\rustyms\rustyms-py\Cargo.toml`

Caused by:
  library target names cannot contain hyphens: rustyms-py
```

Reply:
Alright with me then.